src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -189,10 +189,8 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-branches, too-many
     models = []
     for config in configs["nodes"].values():
         if config["resource_type"] == "model":
-            # conform to the same schema that dbt Cloud uses for models
-            unique_id = config["uniqueId"] = config["unique_id"]
+            unique_id = config["unique_id"]
             config["children"] = configs["child_map"][unique_id]
```

Comment:
Should this `config["children"] = configs["child_map"][unique_id]` be handled as part of the `@pre_load` logic?

Reply:
I considered that, but note that `child_map` lives in the manifest root, not inside the model config, so `pre_load` doesn't have access to it.

Reply:
ohhh, good call! Did not notice that.